pkgs/tools/misc/e/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl }:
+
+let
+  version = "0.02718";
+in
+stdenv.mkDerivation {
+  name = "e-${version}";
+  
+  src = fetchurl {
+    url = "http://www.sourcefiles.org/Productivity_Tools/Calculators/e-${version}.tar.gz";
+    sha256 = "1rxkb0wm136smdn0q359srvw0ppah06441rla7hy1h0k9n0bwawk";
+  };
+
+  buildInputs = [ stdenv ];
+
+  # These morons ship a binary in their tar.gz. What the hell?
+  buildPhase = ''
+    rm e
```

Comment:
You should probably write this is as `preBuild = "rm e";`. Then you can leave the `buildPhase` attribute alone.